app/views/users/show.html.haml

```diff
@@ -0,0 +1,6 @@
+%h5 Welcome
+%h6 User Name
+= @user.name
+, #{@user.email}
```

Comment:
Combine this with teh line above to make it more clear what is going on...

= "#{@user.name}, #{@user.email}"